missingno/missingno.py

```diff
@@ -545,3 +545,27 @@ def geoplot(df,
         plt.show()
     else:
         return ax
+
+
+def pattern(df, counter_name='mvcount', sort=True):
+    """
+    This function will replace the non-missing values with 1 and missing values with 0. The frequency of each pattern will be calculated in a column with the default name 'mvcount'. By default, the resulting dataframe will be sorted in order of descending frequency.
```

Comment:
What is the advantage of this function? Any specific use cases ?

Reply:
I've used it in the past to understand the relative prevalence of specific patterns in missing-ness of data. It's useful for exploratory data analysis.